frontend/src/components/viewers/MetricsVisualizations.tsx

```diff
@@ -130,6 +177,38 @@ function getVerifiedMetricsArtifacts(
   );
 }
 
+function getV1VisualizationArtifacts(
+  linkedArtifacts: LinkedArtifact[],
+  artifactTypes: ArtifactType[],
+): LinkedArtifact | undefined {
+  if (!linkedArtifacts || !artifactTypes) {
```

Comment:
nit: this check is unnecessary, because when you declare the argument types, you didn't allow null/undefined. Typescript will report a type error if inputs could be null/undefined.

If you actually want the function to support undefined inputs, then you should define like:
```
function getV1VisualizationArtifacts(
  linkedArtifacts?: LinkedArtifact[],
  artifactTypes?: ArtifactType[],
)
```

Reply:
Thank you for pointing it out! Fixed. I have a habit of checking parameters are null (in Java).